includes/Classifai/Providers/Azure/Read.php

```diff
@@ -125,7 +125,20 @@ public function read_document() {
 
 		$filesize = filesize( get_attached_file( $this->attachment_id ) );
 		if ( ! $filesize || $filesize > computer_vision_max_filesize() ) {
-			return $this->log_error( new WP_Error( 'size_error', esc_html__( 'Document does not meet size requirements. Please ensure it is smaller than the maximum threshold (defaults to 4MB).', 'classifai' ), $metadata ), $attachment_id );
+			return $this->log_error(
+				new WP_Error(
+					'size_error',
+					esc_html(
+						sprintf(
+							// translators: %1$s is the document file size, %2$s is the current default max filesize
+							__( 'Document (%1$s) does not meet size requirements. Please ensure it is smaller than the maximum threshold (currently %1$s, defaults to 4MB).', 'classifai' ),
+							$filesize,
```

Comment:
While it's extremely unlikely (maybe even impossible due to other checks that happen earlier), in theory `$filesize` could be `false` here, since the `filesize` function can return `false`. I don't believe that will cause an error here, I think it will just cast `false` to a string, which should result in the value being presented as `0`. If we want the message to make more sense, we could do something like:
```suggestion
							! $filesize ? __( 'size not found', 'classifai' ) : $filesize,
```